yt_dlp/extractor/cgtn.py

```diff
@@ -29,8 +29,8 @@ class CGTNIE(InfoExtractor):
                 'title': 'China, Indonesia vow to further deepen maritime cooperation',
                 'thumbnail': r're:^https?://.*\.png$',
                 'description': 'China and Indonesia vowed to upgrade their cooperation into the maritime sector and also for political security, economy, and cultural and people-to-people exchanges.',
-                'author': 'CGTN',
-                'category': 'China',
+                'creator': 'CGTN',
```

Comment:
test will fail w/o `creators`, so might as well fix that while we're at it
```suggestion
                'creators': ['CGTN'],
```